.travis.yml

```diff
@@ -12,7 +12,6 @@ addons:
   google-chrome: latest
 
 install:
-  - npm install npm@5 -g
```

Comment:
Right: https://github.com/vaadin/vaadin-element-skeleton/pull/37

This was the only one experimental component with `npm install npm@5 -g`, that's why I made this change as a PR.